src/notes/notes.service.ts

```diff
@@ -79,7 +88,7 @@ export class NotesService {
    */
   async createNote(
     noteContent: string,
-    alias?: NoteMetadataDto['alias'],
+    alias?: string,
     owner?: User,
   ): Promise<Note> {
     const newNote = Note.create();
```

Comment:
The `create` function of Note also has the alias and and the owner parameter. This should be used here I think.